setup.py

```diff
@@ -30,7 +30,7 @@
 
 setup(
     name=NAME,
-    version="0.0.3",
+    version="0.0.4",
```

Comment:
My normal bump2version didn't work, I don't immediately recognize the config so I did this manually